.github/workflows/pr.yaml

```diff
@@ -16,12 +16,13 @@ jobs:
       - changed-files
       - checks
       - conda-cpp-build
-      - conda-cpp-tests
+      # - conda-cpp-tests
```

Comment:
TODO: add this back before merging

*(just uncommented during because these take 2-3 hours to run, and I wanted to avoid taking up GPU runners for that long on tests that are unlikely to be affected by this PR's changes)*

Reply:
I'd uncomment this as soon as we think this might be merge-ready.

Reply:
re-enabled in https://github.com/rapidsai/cuvs/commit/baae0a7de76021391b195f99b1198f21882f9b87